src/System.IO.FileSystem/tests/Enumeration/AttributeTests.netcoreapp.cs

```diff
@@ -111,5 +112,30 @@ public void DirectoryAttributesAreExpected()
                 Assert.False(enumerator.MoveNext());
             }
         }
+
+        [Fact]
+        public void IsHiddenAttribute()
+        {
+            DirectoryInfo testDirectory = Directory.CreateDirectory(GetTestFilePath());
+            FileInfo fileOne = new FileInfo(Path.Combine(testDirectory.FullName, GetTestFileName()));
+
+            // Put a period in front to make it hidden on Unix
+            FileInfo fileTwo = new FileInfo(Path.Combine(testDirectory.FullName, "." + GetTestFileName()));
+
+            fileOne.Create().Dispose();
+            fileTwo.Create().Dispose();
+            if (PlatformDetection.IsWindows)
+                fileTwo.Attributes = fileTwo.Attributes | FileAttributes.Hidden;
+
+            IEnumerable<string> enumerable = new FileSystemEnumerable<string>(
+                testDirectory.FullName,
+                (ref FileSystemEntry entry) => entry.ToFullPath(),
+                new EnumerationOptions() { AttributesToSkip = 0 })
+            {
+                ShouldIncludePredicate = (ref FileSystemEntry entry) => entry.IsHidden
```

Comment:
`new FileSystemEntry().IsHidden` will null ref.  Do we care?

Reply:
We don't- you can't populate these yourself and we want to discourage newing these up